src/main/java/com/google/sps/servlets/ScheduledInterviewServlet.java

```diff
@@ -232,19 +263,26 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   public void doPut(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String shadowEmail = userService.getCurrentUser().getEmail();
     String shadowId = getUserId();
+
     InterviewPostOrPutRequest putRequest;
     try {
       putRequest = new Gson().fromJson(getJsonString(request), InterviewPostOrPutRequest.class);
-    } catch (JsonSyntaxException jse) {
+    } catch (Exception JsonSyntaxException) {
```

Comment:
Why is this reverted? Please change back to JsonSyntaxException jse

Reply:
Done, happened while resolving conflicts